trace-dispatcher/doc/trace-dispatcher.md

```diff
@@ -108,689 +89,654 @@ data TraceAddBlockEvent blk =
+Similarly, to route a single trace to multiple tracers simultaneously, the fact that `Tracer` is a `Semigroup` allows us to utilize the `<>` operator or `mconcat` for lists of tracers:
 
 ```haskell
 (<>) :: Monoid m => m -> m -> m
 mconcat :: Monoid m => [m] -> m
 ```
 
-In the next example we unite two traces to one tracer, for which we trivially use the same tracer on the right side.
+For instance, to direct messages from one trace to two tracers simultaneously, we can use:
 
 ```haskell
-tracer1  = appendName "tracer1" exTracer
-tracer2  = appendName "tracer2" exTracer
+let resTrace = tracer1 <> tracer2
 ```
 
-## Configuration
-
-The configurability of __dispatchers__ this library allows to define is based on:
-
-1. __Tracer namespace__-based configurability, down to single __message__ granularity,
-2. Runtime reconfigurability, triggered by invocation of `configureTracers`,
-3. Documentation entries __message__.
+## Documentation Generation
 
-Reconfiguration can be triggered at runtime and essentially involves running the entire __dispatcher__ trace network, by doing trace specialisation for each trace that has documentation entries defined.
-
-```haskell
--- The function configures the traces with the given configuration
-configureTracers :: Monad m => TraceConfig -> Documented a -> [Trace m a]-> m ()
-```
+The documentation for tracers is periodically generated and can be accessed in the cardano-node-wiki repository at the following path: [cardano-node-wiki/tracers_doc_generated.md](https://github.com/input-output-hk/cardano-node-wiki/blob/main/docs/new-tracing/tracers_doc_generated.md).
 
-These are the options that can be configured based on a namespace:
+To generate the documentation within GHCi, load the `Cardano.Node.Tracing.Documentation` module and execute the `runTraceDocumentationCmd` function with the appropriate parameters:
 
 ```haskell
-data ConfigOption =
-    -- | Severity level for filtering (default is Warning)
-    ConfSeverity SeverityF
-    -- | Detail level of message representation (Default is DNormal)
-  | ConfDetail DetailLevel
-  -- | To which backend to pass
-  -- Default is [EKGBackend, Forwarder, Stdout HumanFormatColoured]
-  | ConfBackend [BackendConfig]
-  -- | Construct a limiter with name (Text) and limiting to the Double,
-  -- which represents frequency in number of messages per second
-  | ConfLimiter Double
-
-data BackendConfig =
-    Forwarder
-  | Stdout FormatLogging
-  | EKGBackend
-
-data TraceConfig = TraceConfig {
-     -- | Options specific to a certain namespace
-    tcOptions            :: Map.Map NamespaceOuter [ConfigOption]
-     -- | Options for trace-forwarder
-  , ...
-}
-```
-
-If the configuration file is in Yaml format, the following entry means, that by default
-all messages with Info or higher Priority are shown:
+data TraceDocumentationCmd
+  = TraceDocumentationCmd
+    { tdcConfigFile :: FilePath
```

Comment:
It's not mentioned explicitly... is this supposed to be a node config file?

Reply:
Yes, it is a node config file. I added a comment